app/src/main/res/layout/news_article_item_list.xml

```diff
@@ -1,22 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:orientation="vertical"
+    android:orientation="horizontal"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="match_parent"
+    android:paddingBottom="16dp">
 
     <com.android.volley.toolbox.NetworkImageView
```

Comment:
This is great, but you likely would like to show the user that you are loading something, right? A progress bar ("spinner") could be handy. Here is one way to implement that:

https://github.com/YaleSTC/android_testing_training/blob/SynchronizedNetworkImageView/app/src/main/java/com/example/samplegallery/Utilities/CallbackNetworkImageView.java

It allows you to register callbacks on the network image view and then define them with a spinner on top. Once you load (or fail to load) an image, you can have the view change visibilities at will! :-)